stable_baselines3/common/logger.py

```diff
@@ -137,16 +137,22 @@ def write(self, key_values: Dict, key_excluded: Dict, step: int = 0) -> None:
             if excluded is not None and ("stdout" in excluded or "log" in excluded):
                 continue
 
-            if isinstance(value, Video):
+            elif isinstance(value, Video):
                 raise FormatUnsupportedError(["stdout", "log"], "video")
 
-            if isinstance(value, Figure):
+            elif isinstance(value, Figure):
                 raise FormatUnsupportedError(["stdout", "log"], "figure")
 
-            if isinstance(value, Image):
+            elif isinstance(value, Image):
                 raise FormatUnsupportedError(["stdout", "log"], "image")
 
-            if isinstance(value, float):
+            elif isinstance(value, str):
+                value = value.encode("unicode_escape").decode(
```

Comment:
you should maybe define a function as it is re-used multiple times, no?

and it is common to log text with special characters?

Reply:
Since we only use it twice I did not want to clutter up the `KVWriter` class. I can add it if you prefer.

Reply:
yes, please do, and outside the KVWriter probably

Reply:
The main idea behind this is to escape newlines, carriage returns and tabs that may be present in the original text.

Reply:
Moved to function in 474ce91344558b1f21da7a06ec3dcf17563eb616